### ISSUE TITLE
Encoding Issues using Export-NavContainerObjects and Import-ObjectsToNavContainer

### ISSUE THREAD

**Issue by mmenninghaus** [Microsoft/navcontainerhelper#348]
We use several scripts to automatically export CAL code as TXT-files from our local nav containers, push them to a git repository, pull the changes of our teammembers from the repository and import it to our local nav containers again. Everything works fine with your scripts, except if we do not change the code in CSide but on the local file system, e.g. in vs code. Then, especially the german Umlauts (ä, ü, ö) are destroyed. I am not able to find out which encoding the files actually have. I tried countless encodings.
The things is, that some of our scripts automatically change all files and nothing bad happens. For example, one script deletes the `Last Modified` Property in every file after exporting and another brings the `Version List` in our preferred order. We do this by using `Get Content`  and `Set-Content -Encoding Default` in PowerShell 5.1.
Long Story Short: Is there an option to enforce an encoding for export (e.g. utf-8). If not - which encoding is required - we could easily convert our code after export, i.e. before import.

You can reproduce the failure by creating a codeunit in CSide and only putting some Umlauts in a comment. Then export this codeunit using Export-NavContainerObjects, open it in vs code or another Editor, add a line, save and Import it again. Reopening in CSide then shows, depending on the editor and the encoding it assumed, that the Umlauts are no longer readable.

**Comment by freddydk** [comment]
The problem here is, that CSIDE and thus the old txt files doesn't support encoding at all.
Most editors today (incl. vscode) will force an encoding on the file when saving and then you are out of luck.
If you open the file in Notepad++ and after loading select Encoding -> Character Sets and then select the character set of your container (OEM Codepage) - then you can edit the file and save it again.
![image](https://user-images.githubusercontent.com/10775043/52702445-7496c980-2f7c-11e9-993d-f2901b6f16e7.png)

Not sure VS Code supports editing non-encoded files.

**Comment by mmenninghaus** [comment]
Now everything makes sense - thank you!
